### ISSUE TITLE
dCache 9.2 cleaner cell error dumps 

### ISSUE THREAD

**Issue by cfgamboa** [dCache/dcache#7485]
Dear all,

The following error traces have been observed,

```
26 Dec 2023 15:55:45 (cleaner-disk) [] Bug detected
java.util.ConcurrentModificationException: null
	at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1784)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at org.dcache.chimera.namespace.DiskCleaner.runDelete(DiskCleaner.java:111)
	at org.dcache.chimera.namespace.AbstractCleaner.lambda$scheduleCleanerTask$0(AbstractCleaner.java:150)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
26 Dec 2023 15:57:14 (cleaner-disk) [] Bug detected
java.util.ConcurrentModificationException: null
	at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1784)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at org.dcache.chimera.namespace.DiskCleaner.runDelete(DiskCleaner.java:111)
	at org.dcache.chimera.namespace.AbstractCleaner.lambda$scheduleCleanerTask$0(AbstractCleaner.java:150)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

The cleaner configuration:

dcache version: dcache-9.2.6-1.noarch

Cleaner cell topology 3 cleaners,
```
[dcint-core001] (cleaner-disk@dcint-core001Domain) admin > cleaner-disk ha show participants 
Participant{id='cleaner-disk@dcint-core001Domain', isLeader=true}
Participant{id='cleaner-disk@dcint-core002Domain', isLeader=false}
Participant{id='cleaner-disk@dcint-core003Domain', isLeader=false}
```

Configuration
```
[${host.name}Domain/cleaner-disk]
cleaner-disk.limits.threads=20
cleaner-disk.limits.batch-size=1000
cleaner-disk.service.pool.timeout=200
cleaner-disk.limits.period = 10
cleaner-disk.limits.period.unit = MILLISECONDS
cleaner-disk.db.connections.max = 100
cleaner-disk.db.connections.idle = 5
```

thank you.

Carlos

**Comment by lemora** [comment]
Hi Carlos.
This issue has been addressed in 9.2.7.
Cheers
Lea

**Comment by cfgamboa** [comment]
Thank you Lea.
